Gemfile

```diff
@@ -1,11 +1,10 @@
 source 'https://rubygems.org'
 
 gem 'rake', '>= 10.1.1'
-gem 'rspec', '~> 2.99.0'
 gem 'rspec-its'
 gem 'puppet-lint', '>= 0.3.2'
 gem 'rspec-puppet', '>= 1.0.1'
-gem 'puppetlabs_spec_helper', :github => 'jenkins-infra/puppetlabs_spec_helper'
+gem 'puppetlabs_spec_helper',   :require => false
```

Comment:
I don't recall specifically what it was breaking for me at the time, likely beaker, but 0.10.2 is needed for future parser/puppet 4.0.0 anyways.